packages/common/src/ui/components/index.ts

```diff
@@ -10,6 +11,7 @@ export * from './LoadingApp';
 export * from './RemoteComponent';
 export { AppNavLink } from './NavLink';
 export {
+  GlobalStyles,
```

Comment:
@mark-prins `GlobalStyles` is exported here?

Reply:
weird. I did a global search for Global styles and it only found the usage in Viewport.
Fine now though 🤷